WcaOnRails/config/locales/time_will_tell.ru.yml

```diff
@@ -0,0 +1,8 @@
+﻿en:
```

Comment:
Whoops! This needs to be `ru`, right now you're clobbering our English definitions, which is why Travis is failing.